resources/var/tmp/resources/init.groovy.d/scripts/init020configureUpdateSite.groovy

```diff
@@ -24,27 +41,33 @@ def convertNodesToUpdateSites(Object nodes, int parentKeyOffset) {
 		// trim the directory from the nodes key
 		def name = node.key.substring(parentKeyOffset+1);
 		def url = node.value;
-    
+
 		println "found update site: ${name} ${url}";
 		updateSites.add(new hudson.model.UpdateSite(name, url));
 	}
 	return updateSites;
 }
 
 def instance = Jenkins.getInstance();
```

Comment:
This line and the next one should be in the "isConfigured" block.
Currently we read the updateSites even if we will not use them.